Report.md

```diff
@@ -129,29 +120,32 @@ Sprint Backlog | A subset of the Product Backlog for development in a sprint (ma
 Product Increment | A potentially shippable version of the product that builds on the previous increment/version
 Sprint Burndown Chart | A graphical representation of the development team's progress over time (backlog items completed)
 
-
 Workproducts and Models
 =======================
 6. Document your proposed workproducts and models *and descriptions*.
 
 ## **Use Case Diagrams**
+These diagrams help to visualise scenarios where there is interaction between users, the system, and other actors (namely Relationship Managers), describing what the system will do but not how it will be done. The diagrams below show this in regards to outbound and inbound calls respectively:
 ### Use Case Diagram - Outbound Calls
 ![Image of Use Case Diagram (Outbound Call)](./diagrams/UseCaseDiagram-OutboundCall.png)
 
 ### Use Case Diagram - Inbound Calls
 ![Image of Use Case Diagram (Inbound Call)](./diagrams/UseCaseDiagram-InboundCall.png)
 
 ## **Activity Diagrams**
+These diagrams help to visualise the behaviour of the system and actors across multiple use cases, including decisions and the flow of what happens, still not describing the how. The diagrams below show this in regards to outbound and inbound calls respectively:
 ### Activity Diagram - Outbound Calls
 ![Image of Travel Company Activity Diagram (Outbound Call)](./diagrams/ActivityDiagram-OutboundCall.png)
 
 ### Activity Diagram - Inbound Calls
 ![Image of Travel Company Activity Diagram (Inbound Call)](./diagrams/ActivityDiagram-InboundCall.png)
 
 ## **Class Diagram**
+This diagram represents the structure of the classes involved in the wider system. It displays their attributes and methods as well as the relationships that are between them, such as below:
 ![Image of Class Diagram](./diagrams/ClassDiagram.png)
 
 ## **Collaboration Diagrams**
+These diagrams represents the dynamic behaviour of objects and the structure of the interactions between them. It also displays the sequence of these interactions, though less than sequence diagrams. The diagrams below show this in regards to outbound and inbound calls respectively:
```

Comment:
Maybe instead of saying "though less than sequence diagrams", maybe say "It also displays the sequence of these interactions through numbering, however focuses more on the messages that flow between each object rather than time-based interactions (seen in sequences diagrams)". Maybe something along those lines?

Reply:
Hey Zac I have also added a description for the Customer Collaboration diagram, but maybe we can use both.

Reply:
Hey Zac I have also added a description for the Customer Collaboration diagram, but maybe we can use both.